### ISSUE TITLE
The getting started guide for running Kubernetes locally via Docker is broken

### ISSUE THREAD

**Issue by roberthbailey** [GoogleCloudPlatform/kubernetes#7292]
As reported on [stack overflow](http://stackoverflow.com/questions/29840709/is-there-a-reliable-way-how-to-run-virtualized-docker-kubernetes-stack-on-wind) by someone who was trying to get it working on Windows. I reproduced the same error on my mac. 

#7077 updated the instructions to v0.15.0 but it have the intended effect, because [cluster/images/hyperkube/master.json](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/images/hyperkube/master.json) and [cluster/images/hyperkube/master-multi.json](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/images/hyperkube/master-multi.json) both point to hyperkube v0.14.1. 

When I walk through the single node example, at the end I see

```
$ kubectl get nodes
NAME        LABELS        STATUS
127.0.0.1   Schedulable   <none>    NotReady
```

instead of a Ready node. 

These manifests are built into the hyperkube v0.15.0 image, and I'm not sure if building a new image with the same tag and pushing it will fix it or if we need to build a hyperkube v0.15.0.1 image so we have a different version number.

**Comment by analytik** [comment]
When I run the commands from master branch today -
```bash
docker run --net=host -d kubernetes/etcd:2.0.5.1 /usr/local/bin/etcd --addr=127.0.0.1:4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
docker run --net=host -d -v /var/run/docker.sock:/var/run/docker.sock  gcr.io/google_containers/hyperkube:v0.14.2 /hyperkube kubelet --api_servers=http://localhost:8080 --v=2 --address=0.0.0.0 --enable_server --hostname_override=127.0.0.1 --config=/etc/kubernetes/manifests
docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v0.14.2 /hyperkube proxy --master=http://127.0.0.1:8080 --v=2
```
I get status Ready, and predictably, kubelet and proxy are running v0.14.2, and controll/schedule/apiserve are v0.14.1.

I still can't get the example to run, and get this error:
```bash
$ kubectl run-container nginx --image=nginx --port=80
Error from server: the server could not find the requested resource (post replicationControllers)
```

**Comment by skuzye** [comment]
Seems like the docs are wrong, as per #7581

**Comment by brendandburns** [comment]
This is all fixed now, verified w/ 1.0.1, closing.
